Source/Dafny/Resolver.cs

```diff
@@ -10,6 +10,7 @@
 using System.Numerics;
 using System.Diagnostics.Contracts;
 using System.Linq.Expressions;
+using System.Security.Cryptography;
```

Comment:
Is this necessary? (I'm assuming Rider inserted it from some temporary typo in the source.)
```suggestion
```